AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Edit.cshtml

```diff
@@ -20,6 +20,32 @@
     }
 }
 
+@section styles{
+    <style>

```

Comment:
Please add these styles to a new style sheet and include them in this section as a link. We're going to likely end up converting to SCSS, and will eventually just be bundling/minifying all the styles.